### DISCUSSION TITLE
Sweep Printout

### DISCUSSION THREAD

**Discussion by cfengno1** [block-hczhai/block2-preview#147] [Q&A] [answered]
Dear Huanchen,

Can you explain what are the meaning of data and cpsd in the printout after each sweep?
``
 | data = 217 GB | cpsd = 172 GB
``

Best,
Feng

**Comment by hczhai** [comment, marked as answer]
The "data" number is the raw scratch data size (per sweep per MPI processor) before compression. The "cpsd" number is the data size written into disk (per sweep per MPI processor) after compression. When you have only one MPI processor, the "cpsd" number corresponds to the leading cost of disk (scratch) space usage.

The compression ratio can be adjusted indirectly via the "fp_codec_cutoff" argument in ``DMRGDriver``. The default setting is lossy compression which provides sufficient accuracy and economic usage of disk space for most applications. Setting "fp_codec_cutoff=0.0" will enable lossless compression of scratch data.
